sdk/ai/azure-ai-projects/samples/agents/sample_agents_openapi_morningstar.py

```diff
@@ -0,0 +1,191 @@
+# pylint: disable=line-too-long,useless-suppression
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+
+"""
+DESCRIPTION:
+    This sample demonstrates how to use agent operations with the
+    Morningstar OpenAPI tool, (https://www.morningstar.com/) from the Azure Agents service using a synchronous client.
+    To learn more about OpenAPI specs, visit https://learn.microsoft.com/openapi
+
+USAGE:
+    python sample_agents_openapi_morningstar.py
+
+    Before running the sample:
+
+    pip install azure-ai-projects azure-identity jsonref
+
+    Set these environment variables with your own values:
+    1) PROJECT_CONNECTION_STRING - The project connection string, as found in the overview page of your
+       Azure AI Foundry project.
+    2) MODEL_DEPLOYMENT_NAME - The deployment name of the AI model, as found under the "Name" column in
+       the "Models + endpoints" tab in your Azure AI Foundry project.
+    3) PROJECT_CONNECTION_STRING - The project connection string of your AI Foundry project.
```

Comment:
3 is a repeat of 1

Reply:
Oops sorry this had to be called PROJECT_CONNECTION_ID, I've fixed it and tested my code again.